graphs/maximum_bipartite_matching.py

```diff
@@ -0,0 +1,64 @@
+class MaximalBipartiteMatching:
+    """
+    This class implements finding maximal Bipartite matching using DFS.
+
+    Usage:
+    >>> bipartite_graph = [
+    ...     [0, 1, 1, 0, 0, 0],
+    ...     [1, 0, 0, 1, 0, 0],
+    ...     [0, 0, 1, 0, 0, 0],
+    ...     [0, 0, 1, 1, 0, 0],
+    ...     [0, 0, 0, 0, 0, 0],
+    ...     [0, 0, 0, 0, 0, 1]
```

Comment:
We should need to add a couple more doctests ?

Reply:
Hi @imSanko I have added more test case please review now.

Reply:
Nice work ✅

Reply:
@imSanko Thank you for approving, 
I am also interested in solving other open issues and working on it.

Reply:
@tianyizheng02  can you review the PR? I have made the changes suggested by @imSanko

Reply:
@imSanko it is still showing "imSanko was requested for review" Can you please help to Review and merge it.